README.md

```diff
@@ -29,102 +29,38 @@ allow consumer, consumer delegates to access the resource.
 - Integration with auditing server for metering purposes
 
 
+## Users and Roles
+
+<div style="text-align: center;">
+<img src="./docs/users-and-roles.png" alt="Users and Roles" width="600" height="400"/>
+</div>
+
+- Provider, Consumers and delegates of provider and consumers are allowed to access the APIs
+- Provider / provider delegate specific APIs : Create Policy, Update access request, Delete Policy
+- Consumer / consumer delegate specific APIs : Create Access Request, Withdraw access request
+- Commonly accessible APIs for both the users : List Access Request,  Get Policies
+- How is the user considered as a consumer, provider or delegate?
+   - If the **role** in token fetched from Auth Server is **provider**, **consumer**
```

Comment:
- I'm unsure that you need to specify how a user is considered to be a consumer/provider etc by making references to the values in the token. 
- Maybe leave it like this for now, but link to the create token API docs because the token fields are specified there (use production link)

Reply:
Thanks for reviewing the PR, I've updated the files [here](https://github.com/datakaveri/iudx-acl-apd/pull/136/commits/bf1a099b1324350fdd6dd214b4372a8743b4b33a)